test/e2e/tree_test.go

```diff
@@ -35,7 +35,7 @@ var _ = Describe("Podman image tree", func() {
 
 	It("podman image tree", func() {
 		SkipIfRemote("Does not work on remote client")
-		dockerfile := `FROM docker.io/library/busybox:latest
+		dockerfile := `FROM quay.io/libpod/busybox:latest
```

Comment:
I wonder if we don't have builds that are just using shortnames, not pre-pulled images.

If we completely rip `docker.io` out of registries.conf, I wonder how much breaks...